lldb/source/Utility/DiagnosticsRendering.cpp

```diff
@@ -102,7 +102,7 @@ void RenderDiagnosticDetails(Stream &stream,
   // characters.  In the future it might make sense to move this into
   // Host so it can be customized for a specific platform.
```

Comment:
Does it make sense to move this into Host now? I'm generally not a fan of `ifdef` code in Utility if it can be helped, but I understand if it would be a large amount of work otherwise.